tests/relationships/test_association_table_relations.py

```diff
@@ -17,8 +17,8 @@ class PublishedArticle(self.Model):
 
             article_id = sa.Column(sa.Integer, sa.ForeignKey('article.id'))
             author_id = sa.Column(sa.Integer, sa.ForeignKey('author.id'))
-            author = relationship('Author')
-            article = relationship('Article')
+            author = relationship('Author',viewonly=True)
+            article = relationship('Article',viewonly=True)
```

Comment:
I see - so, this relationship is conflicting with `Author.articles` becuase both of the relationships set the foreign keys.
To retain the previous behavior, I think you shold mark `overlaps='articles'` instead of `viewonly`
Because earlier the test allowed the developer to use either relationship and both used to set the foreign key.

The overlaps just lets sqlalchemy know that we are aware that there is an overlap and it's not a bug

Also - there should be a space after the comma I believe to be consistent with the styling of the project

Reply:
Hi @AbdealiJK ,

Made the suggested changes